src/core/worker.c

```diff
@@ -101,29 +95,36 @@ QuicWorkerInitialize(
     Worker->ExecutionContext.NextTimeUs = UINT64_MAX;
     Worker->ExecutionContext.Ready = TRUE;
 
-#ifdef QUIC_USE_EXECUTION_CONTEXTS
-    UNREFERENCED_PARAMETER(ThreadFlags);
-    CxPlatAddExecutionContext(&Worker->ExecutionContext, IdealProcessor);
-#else
-    CXPLAT_THREAD_CONFIG ThreadConfig = {
-        ThreadFlags,
-        IdealProcessor,
-        "quic_worker",
-        QuicWorkerThread,
-        Worker
-    };
-
-    Status = CxPlatThreadCreate(&ThreadConfig, &Worker->Thread);
-    if (QUIC_FAILED(Status)) {
-        QuicTraceEvent(
-            WorkerErrorStatus,
-            "[wrkr][%p] ERROR, %u, %s.",
-            Worker,
-            Status,
-            "CxPlatThreadCreate");
-        goto Error;
+#ifndef _KERNEL_MODE // Not supported on kernel mode
+    if (ExecProfile != QUIC_EXECUTION_PROFILE_TYPE_MAX_THROUGHPUT) {
```

Comment:
Does max throughput related flag works without splitpartitioning?

Reply:
Yes. Max throughput just means separate threads for UDP and QUIC now.